.github/workflows/jwa_manifests_build.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build JWA manifests
+on:
+  pull_request:
+    paths:
+      - components/crud-web-apps/jupyter/**
+      - components/crud-web-apps/common/**
```

Comment:
Here we don't even need to include the common code at all, since we care about the manifests.

So in this case it should only be:
`components/crud-web-apps/jupyter/manifests/**`

Reply:
I deleted the common code from the paths.